src/Symfony/Component/Translation/Dumper/XliffFileDumper.php

```diff
@@ -36,6 +48,10 @@ public function dump(MessageCatalogue $messages, $options = array())
             $this->defaultLocale = \Locale::getDefault();
         }
 
+        if (array_key_exists('tool_info', $options)) {
+            $this->toolInfo = array_merge($this->toolInfo, $options['tool_info']);
```

Comment:
this is not a good idea. You should not change the state of the dumper as it will affect subsequent dumps too